Cargo.toml

```diff
@@ -52,7 +53,10 @@ rayon = "1.3"
 serde = "1.0.166"
 serde_derive = "1.0.166"
 serde_json = { version = "1" }
-wasmtime = { version = "12.0.0", default-features = false, features = ['cranelift', 'component-model'] }
+wasmtime = { version = "12.0.0", default-features = false, features = [
+  'cranelift',
+  'component-model',
+] }
```

Comment:
Would it be ok to back out some of the formatting-related changes in this file?

Reply:
Will do.
Was probably just cargo fmt.

Reply:
Done